tests/Microsoft.Identity.Test.Unit/WwwAuthenticateParametersTests.cs

```diff
@@ -253,6 +286,60 @@ public void ExtractClaimChallengeFromHeader_IncorrectError_ReturnNull()
             Assert.IsNull(WwwAuthenticateParameters.GetClaimChallengeFromResponseHeaders(httpResponse.Headers));
         }
 
+        [TestMethod]
+        public async Task CreateAllFromResourceResponseAsync_HttpClient_Bearer_Pop_Async()

```

Comment:
Can you measure the code coverage on the new code? I would expect this to be fully (100%) covered.

Reply:
I did this and the results are good. I think the only untested block of code is 

<img width="889" alt="image" src="https://user-images.githubusercontent.com/12273384/207066752-4c74846b-4880-4e29-82a4-56196bee27c9.png">


To do this yourself, right click on the test class and select "analyze test coverage"

<img width="271" alt="image" src="https://user-images.githubusercontent.com/12273384/207066919-174b6dca-d463-4ea1-b5ec-e6df82c9cf68.png"> and then just navigate to the product code and you'll see it highlighted. Missing coverage is in red, good coverage in blue.

Reply:
It is actually kinda difficult to create a test that throws an exception. Whenever I try to create a header with a malformed value, the .NET code throws an exception before the test method has a chance to execute. ill have to keep playing around with it.